db/schema.rb

```diff
@@ -158,120 +158,6 @@
-    t.float "p911_tuition_fees"
-    t.integer "p911_recipients"
-    t.float "p911_yellow_ribbon"
-    t.integer "p911_yr_recipients"
-    t.boolean "dodmou"
-    t.string "insturl"
-    t.integer "pred_degree_awarded"
-    t.integer "locale"
-    t.integer "undergrad_enrollment"
-    t.float "retention_all_students_ba"
-    t.float "retention_all_students_otb"
-    t.float "graduation_rate_all_students"
-    t.float "transfer_out_rate_all_students"
-    t.float "salary_all_students"
-    t.float "repayment_rate_all_students"
-    t.float "avg_stu_loan_debt"
-    t.boolean "credit_for_mil_training"
-    t.boolean "vet_poc"
-    t.boolean "student_vet_grp_ipeds"
-    t.boolean "soc_member"
-    t.string "calendar"
-    t.boolean "online_all"
-    t.string "vet_tuition_policy_url"
-    t.integer "tuition_in_state"
-    t.integer "tuition_out_of_state"
-    t.integer "books"
-    t.boolean "sec_702"
-    t.boolean "caution_flag"
-    t.text "caution_flag_reason"
-    t.integer "complaints_facility_code"
-    t.integer "complaints_financial_by_fac_code"
-    t.integer "complaints_quality_by_fac_code"
-    t.integer "complaints_refund_by_fac_code"
-    t.integer "complaints_marketing_by_fac_code"
-    t.integer "complaints_accreditation_by_fac_code"
-    t.integer "complaints_degree_requirements_by_fac_code"
-    t.integer "complaints_student_loans_by_fac_code"
-    t.integer "complaints_grades_by_fac_code"
-    t.integer "complaints_credit_transfer_by_fac_code"
-    t.integer "complaints_job_by_fac_code"
-    t.integer "complaints_transcript_by_fac_code"
-    t.integer "complaints_other_by_fac_code"
-    t.integer "complaints_main_campus_roll_up"
-    t.integer "complaints_financial_by_ope_id_do_not_sum"
-    t.integer "complaints_quality_by_ope_id_do_not_sum"
-    t.integer "complaints_refund_by_ope_id_do_not_sum"
-    t.integer "complaints_marketing_by_ope_id_do_not_sum"
-    t.integer "complaints_accreditation_by_ope_id_do_not_sum"
-    t.integer "complaints_degree_requirements_by_ope_id_do_not_sum"
-    t.integer "complaints_student_loans_by_ope_id_do_not_sum"
-    t.integer "complaints_grades_by_ope_id_do_not_sum"
-    t.integer "complaints_credit_transfer_by_ope_id_do_not_sum"
-    t.integer "complaints_jobs_by_ope_id_do_not_sum"
-    t.integer "complaints_transcript_by_ope_id_do_not_sum"
-    t.integer "complaints_other_by_ope_id_do_not_sum"
-    t.float "retention_rate_veteran_ba"
-    t.float "retention_rate_veteran_otb"
-    t.float "persistance_rate_veteran_ba"
-    t.float "persistance_rate_veteran_otb"
-    t.float "graduation_rate_veteran"
-    t.float "transfer_out_rate_veteran"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["cross"], name: "index_data_csvs_on_cross"
-    t.index ["facility_code"], name: "index_data_csvs_on_facility_code", unique: true
-    t.index ["institution"], name: "index_data_csvs_on_institution"
-    t.index ["ope"], name: "index_data_csvs_on_ope"
-  end
```

Comment:
What is going on with these tables (and `arf_gibills`)? As far as I can tell none of them _should_ exist.

There have been 9 commits since Dec 4 adding and removing these tables.

We need to get everyone on the same page.

OK, I think I know what is going on:
`data_csvs`: the migration to create this table was deleted [here](https://github.com/department-of-veterans-affairs/gibct-data-service/commit/b5247e52c8517dbd3e96a83eef27fba27d3ae454)
instead of creating a migration to drop the table.  That would explain why it existed in `production` but not necessarily in master's `schema.rb`
[This commit](https://github.com/department-of-veterans-affairs/gibct-data-service/commit/4c0da20d61ea5e69fac87e22012c3230f0a8142a) contains the deletion of migrations that created the `arf_gibills`, `csv_storages`, `csv_files` and `va_crosswalks` tables.  Again, there is no migration to drop the tables.
That is why these tables exist on production (and are not used), and shouldn't be in `schema.rb`
cc: @annaswims

Reply:
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/511 changed the schema to match what is in staging and production

`arf_gi_bills` has a model object and is shown "Latest Uploads" table in GIDS as CSV that can be uploaded 

`csv_files`, `csv_storages`, `data_csvs`, `va_crosswalks` do not have corresponding model objects

Reply:
RIght.  `arf_gi_bills` is good. `arf_gibills` is bad.

Reply:
https://github.com/department-of-veterans-affairs/gibct-data-service/pull/525 resolved this issue